packages/select/src/components/query-list/queryList.tsx

```diff
@@ -561,14 +561,14 @@ export class QueryList<T> extends AbstractComponent2<QueryListProps<T>, IQueryLi
         return getFirstEnabledItem(this.state.filteredItems, this.props.itemDisabled, direction, startIndex);
     }
 
-    private isCreateItemRendered(): boolean {
+    private isCreateItemRendered(createNewItem?: T): boolean {
```

Comment:
can you please leave a `/** jsdoc */` comment on this method explaining briefly why we need this argument?

Reply:
All set 👍🏻